### PR TITLE
Scry chat

### PR DESCRIPTION
**Pull request by jack829**

### CODE REVIEW

File: src/screens/Chat.js [hunk 1]
```diff
@@ -10,35 +11,99 @@ import {
...
+    const {
+      curated: chatAndSuggestionsCurated
+    } = chatAndSuggestions.reduce((accumulator, suggestion, index) => {
+      const { status } = suggestion
+      const isSuggestion = !!status
+      let newCurrentSuggestion = false
+
+      if (isSuggestion) {
+        // If the current message is a suggestion, arrange the aggregate `currentSuggestion` property
+        //  to begin tallying all sequential suggestions
```
Comment (dblanchard13) [inline src/screens/Chat.js]: Is the extra space to start this line intentional?

File: src/screens/Chat.js [hunk 2]
```diff
@@ -63,13 +128,11 @@ class Chat extends Component {
 
   addCallbackToSocket() {
     this._callbackID = SocketInterface.addCallback(function(message) {
-      console.log(' ')
-      console.log('in Chat component callback')
-      console.log(' ')
-      let { messages } = this.state
-
-      this.setState({ messages: messages.slice() })
+      // console.log(' ')
```
Comment (dblanchard13) [inline src/screens/Chat.js]: Can you just get rid of these console.logs?

File: src/screens/TinderChat/TinderChat.js [hunk 3]
```diff
@@ -44,15 +42,7 @@ class Chat extends Component {
...
-    } = conversation.Suggestions.reduce((accumulator, suggestion) => {
-      const { status } = suggestion
-      if (accumulator[status]) {
-        accumulator[status].push(suggestion)
-      } else if (status) {
-        accumulator[status] = [suggestion]
-      }
-      return accumulator
-    }, {})
+    } = splitSuggestions(conversation.Suggestions)
```
Comment (dblanchard13) [inline src/screens/TinderChat/TinderChat.js]: I like this 🤓

File: src/components/MessageSender.js [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+    justifyContent: 'center',
+    alignSelf: 'flex-end',
+    ...THUMBNAIL_PIC,
+  },
+  name: {
+    textAlign: 'center',
+  }
+})
+
+export default messageSender;
```
[the remark this answers is not in the document]
Comment (jack829) [reply src/components/MessageSender.js]: oh yeah, i use a code snippet thing to generate files which automatically puts them in. Just missed this one (and probably some more)

File: src/screens/Chat.js (hunk 2 above)
Comment (jack829) [reply src/screens/Chat.js]: ha, yeah I thought you'd left them on purpose for tracking that

File: src/components/MessageSender.js [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+} from 'react-native'
+import AppText from './AppText'
+import {
+  THUMBNAIL_PIC,
+} from '../globals/styles'
+
+const messageSender = (props) => (
+  <View style={styles.container}>
+    <AppText style={styles.name}>
+      {//User.first_name.length > 5 ? User.first_name.substring(0, 5) + '...' : User.first_name
```
[the remark this answers is not in the document]
Comment (jack829) [reply src/components/MessageSender.js]: nope, just didn't look back at this file

File: src/screens/Chat.js (hunk 1 above)
Comment (jack829) [reply src/screens/Chat.js]: Oh, yeah i think? I'm inconsistent with that. I'll just leave it flat

File: src/screens/TinderChat/TinderChat.js (hunk 3 above)
Comment (jack829) [reply src/screens/TinderChat/TinderChat.js]: Ha, yeah I thought I was going to reuse it but then I didn't. So might not need it, but i still like smaller files

**Pull request merged**